app/pages/[teamSlug]/customers/index.page.tsx

```diff
@@ -22,18 +35,25 @@ const CustomersPage = (): JSX.Element => {
           </Button>
         </div>
 
-        <table className="w-full">
-          <tr>
-            <th>Name</th>
-            <th>Customer-ID</th>
-          </tr>
+        <Table className="w-full">
+          <TableHeadRow>
+            <TableHeadCell>Name</TableHeadCell>
+            <TableHeadCell>Customer-ID</TableHeadCell>
+          </TableHeadRow>
           {data?.teamBySlug.customers.map((customer) => (
-            <tr key={customer.id}>
-              <td>{customer.title}</td>
-              <td>{customer.id}</td>
-            </tr>
+            <TableBody key={customer.id}>
```

Comment:
The `.map` should return a `<TableRow>` for every customer.
The `<TableBody>` has to be outside the `.map()`